setup.sh

```diff
@@ -46,13 +88,31 @@ while [ "$OUTPUT" -ne 200 ] && [ "$TIMEOUT" -gt 0 ]
     done
 
 if [ "$TIMEOUT" -le 0 ]; then
-    echo "\n${RED}Failed to start Cloudant service. Terminating setup.${PLAIN}\n"
+    printf "\n\n${CYAN}Status: ${PLAIN}${RED}Failed to start Cloudant service. Terminating setup.${PLAIN}\n\n"
     exit 1
-else
-    echo "\n${CYAN}Cloudant started.${PLAIN}\n"
+fi
+printf "\n${CYAN}Cloudant started.${PLAIN}\n"
 
-    ## create database --- TODO check return code 
-    echo "\n${CYAN}Creating database in Cloudant${PLAIN}\n"
-    curl --request PUT --url http://admin:pass@0.0.0.0:8080/test-db
+## create database
+printf "\n${RED}>> Creating database in Cloudant${PLAIN}"
+curl --request PUT --url http://$USER:$PASSWORD@$HOST:$PORT/$DATABASE > /dev/null 2>&1
+DB_OUTPUT=$?
+if [ "$DB_OUTPUT" -ne 0 ]; then
+    printf "\n\n${CYAN}Status: ${PLAIN}${RED}Database could not be created. Terminating setup.${PLAIN}\n\n"
+    exit 1
 fi
+printf "\n${CYAN}Database created succesfully.${PLAIN}\n"
+
+## set env variables for running test
+printf "\n${RED}>> Setting env variables to run test${PLAIN} ${GREEN}...${PLAIN}"
+export CLOUDANT_URL=http://$USER:$PASSWORD@$HOST:$PORT
```

Comment:
It is setup in this way so the user can run the tests after just sourcing the script. `source setup.sh` and then `npm test`. If they just run the script by itself, they need to set the env variables by themselves.